plugins/ls-bayesian-agent/src/main/resources/com.redhat.bayesian.lsp.script.sh

```diff
@@ -0,0 +1,155 @@
+unset PACKAGES
+unset SUDO
+unset PAYLOAD
+
+command -v bzip2 >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" bzip2"; }
+command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
+command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
+
+test "$(id -u)" = 0 || SUDO="sudo"
```

Comment:
@sunix That won't work on OpenShiftOnline (`sudo` is not allowed). Please refer to agents scripts in `openshift-connector` branch (for example https://github.com/eclipse/che/blob/openshift-connector/agents/ls-typescript/src/main/resources/org.eclipse.che.ls.typescript.script.sh#L16).